### PR TITLE
Fix panic when perspective property is zero.

### PR DESCRIPTION
**Pull request by glennw**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9446)

### CODE REVIEW

File: tests/wpt/mozilla/tests/css/perspective_zero.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
```
Comment (jdm) [inline tests/wpt/mozilla/tests/css/perspective_zero.html]: Missing `<link>` to identify the reference file.

Comment (glennw) [inline tests/wpt/mozilla/tests/css/perspective_zero.html]: Updated - although it was definitely running that test locally without that line. Just out of curiosity, why do we need to list it both there are also (twice) in the manifest file?

**Comment by jdm** [comment]
The manifest file is generated automatically by extracting the metadata from the test files. Hand-editing MANIFEST.json is an anti-pattern; it should be updated via `./mach create-wpt` or `./mach test-wpt --manifest-update`.

**Comment by glennw** [comment]
@bors-servo retry

(Seems infrastructure related - very large number of unrelated tests failed, which also has occurred to several other builds on this machine today).

**Pull request merged**